### PR TITLE
chore: Add workflows

### PR DESCRIPTION
**Pull request by andykog**
Add GH workflows:
- test - run tests on PR's & commits in master
- release - can be triggered manually to create a new GH release with attached files chrome.zip & firefox.zip. Commit SHA is used as release name
- mobx-devtools-mst-publish - publish mobx-devtools-mst npm package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build:chrome": "cross-env NODE_ENV=production cross-env NODE_OPTIONS='--openssl-legacy-provider' TARGET_BROWSER=chrome node scripts/webextension/build.js && echo 'Chrome built'",
     "start:firefox": "cross-env NODE_ENV=development cross-env TARGET_BROWSER=firefox NODE_OPTIONS='--openssl-legacy-provider' node scripts/webextension/dev-server.js",
     "build:firefox": "cross-env NODE_ENV=production TARGET_BROWSER=firefox NODE_OPTIONS='--openssl-legacy-provider' node scripts/webextension/build.js",
-    "test": "npm run test:unit ; npm run test:e2e",
+    "test": "npm run test:unit",
```
Comment (andykog) [inline package.json:14]: e2e tests are broken, excluded them from running in CI

Comment (coolsoftwaretyler) [reply package.json:14]: @andykog - I think only Firefox e2e is broken. Maybe worthwhile to change this to test:e2e:chrome to at least run something.

Those tests don't do much, so I don't feel very strongly about it one way or another.

**Comment by coolsoftwaretyler** [comment]
@andykog this is great, thank you very much! I think you're good to merge, but I had one suggestion on running at least Chrome e2e tests. I don't feel strongly so I say merge either way you prefer.

File: package.json (hunk 1 above)
Comment (andykog) [reply package.json:14]: For some reason chrome seems to hang: https://github.com/mobxjs/mobx-devtools/actions/runs/11405976300/job/31738662591
Maybe there's some problem with the invironment
Merging like this for now

**Pull request merged**